web/src/lib/components/pages/record.tsx

```diff
@@ -15,6 +15,16 @@ import confirm from '../confirm';
 const CACHE_SET_COUNT = 9;
 const SET_COUNT = 3;
 const PAGE_NAME = 'record';
+const MIN_RECORDING_LENGTH = 400;   // ms
```

Comment:
Nice. Let's change this to 300 milliseconds, as we could have some 2 syllable sentences (eg. "hi there"), and according to this, unstressed syllables are 150ms long.

https://www.uni-bamberg.de/fileadmin/eng-ling/fs/Chapter_13/Index.html?22Theacousticmethod.html